aws/scripts/py/cache_invalidation.py

```
@@ -0,0 +1,97 @@
+                    print(f"Found staging distribution: {dist['Id']}")
+                else:
+                    production_dist = dist
+                    print(f"Found production distribution: {dist['Id']}")
+                break
+
+    if not staging_dist or not production_dist:
+        raise Exception("Could not find both staging and production distributions for vilnacrmtest.com")
+    
+    return staging_dist, production_dist
+
+def invalidate_cache(distribution, is_staging=True):
```

Ruff (0.11.9)

Missing return type annotation for public function `invalidate_cache`

(ANN201)